msteams-platform/samples/virtual-assistant.md

```diff
@@ -320,17 +325,19 @@ Some messaging extension activities do not include the command ID. For example,
 
 ---
 
-## Example: Convert the Book-a-room app template to a Virtual Assistant skill
+## Example
```

Comment:
Example as a standalone will confuse in heading level 2. Can you make the heading as Example of ...